package.json

```diff
@@ -3,7 +3,15 @@
   "version": "1.0.0",
   "description": "https://firewaterapparel.co",
   "scripts": {
-    "start": "http-server -p 8080"
+    "start": "http-server -p 8080 ./dist",
+    "format:staged": "pretty-quick --staged",
+    "prepare": "husky install",
+    "build": "broccoli build dist"
```

Comment:
run things in `package.json->scripts` with `npm run`, e.g. `npm run build`.

`prepare` is ran automatically by npm when doing `npm install` to install dependencies. `format:staged` is called by husky (a dependency to manage git hooks) on commit to ensure committed code is formatted.